docs/cli/gittuf.md

```diff
@@ -17,6 +17,7 @@ A security layer for Git repositories, powered by TUF
 * [gittuf add-hooks](gittuf_add-hooks.md)	 - Add git hooks that automatically create and sync RSL
 * [gittuf clone](gittuf_clone.md)	 - Clone repository and its gittuf references
 * [gittuf dev](gittuf_dev.md)	 - Developer mode commands
+* [gittuf gittuf](gittuf_gittuf.md)	 - applies work in progress changes to the policy state to the current policy state
```

Comment:
Looks like a mistake but this command is plumbed through as gittuf gittuf apply?

Reply:
I have fixed this, but that is what `make generate`. After these changes if I run `make generate`, the changes are reversed and the `gittuf gittuf` command is added to the docs

Reply:
Docs are failing, since generate creates a different set of docs with the `gittuf gittuf` command